src/taipy/rest/api/resources/job.py

```diff
@@ -157,22 +169,22 @@ def get(self):
     @_middleware
     def post(self):
         args = request.args
-        task_name = args.get("task_name")
+        task_config_id = args.get("task_id")
 
-        if not task_name:
-            return {"msg": "Config id is mandatory"}, 400
+        if not task_config_id:
+            return {"msg": "Task config_id is mandatory"}, 400
```

Comment:
Message is strange: the parameter is 'task_id', and you complain about 'config_id'

Reply:
The original naming of this API parameter is _task_id, but it is actually task _config_id_